src/nvim/tui/tui.c

```diff
@@ -100,7 +100,7 @@ typedef struct {
   bool immediate_wrap_after_last_column;
   bool bce;
   bool mouse_enabled;
-  bool busy, is_invisible;
+  bool busy, is_invisible, should_be_invisible;
```

Comment:
conventional name (in this codebase) would be:
```suggestion
  bool busy, is_invisible, want_invisible;
```

Reply:
This was marked as resolved but no change was pushed.

Reply:
I solved it locally but hadn't pushed yet. I will wait next time befire marking it as pushed.